data-raw/process_data_sets_maggie.R

```diff
@@ -0,0 +1,266 @@
+riddler_pick_lowest <- clean_names(riddler_pick_lowest)
+devtools::use_data(riddler_pick_lowest, overwrite = TRUE)
+
+# sandy-311-calls---------------------------------------------------------------------
+sandy_311 <- read_csv("data-raw/sandy-311-calls/sandy-311-calls-by-day.csv")
+sandy_311 <- clean_names(sandy_311)
+
+sandy_311 <- sandy_311 %>%
+  rename(
+    nyc_311 = nyc_3_1_1,
+    nyc_service = nycservice,
+    nys_emergency_mg = nysemergencymg
+  ) %>%
+  mutate(
+    date = as.Date(date, format= "%m / %d / %y")
+  )
+sandy_311<-gather(sandy_311, "agency", "num_calls", 2:24)
+sandy_311$agency <- as.factor(sandy_311$agency)
+devtools::use_data(sandy_311, overwrite = TRUE)
+
+# trump-approval-ratings---------------------------------------------------------------------
+trump_approval_poll <- read_csv("data-raw/trump-approval-ratings/approval_polllist.csv")
+trump_approval_poll <- trump_approval_poll %>%
+  mutate(
+    multiversions = ifelse(multiversions == "*", TRUE, FALSE),
+    multiversions = ifelse(is.na(multiversions), FALSE, TRUE),
+    tracking = ifelse(is.na(tracking), FALSE, TRUE),
+    subgroup = as.factor(subgroup),
+    modeldate = as.Date(modeldate, format = "%m / %d / %Y"),
+    startdate = as.Date(startdate, format = "%m / %d / %Y"),
+    enddate = as.Date(enddate, format = "%m / %d / %Y"),
+    pollster = as.factor(pollster),
+    grade = as.factor(grade),
+    population = as.factor(population),
+    url = as.factor(url),
+    createddate = as.Date(createddate, format = "%m / %d / %Y")
+  ) %>%
+  rename(
+    model_date = modeldate,
+    start_date = startdate,
+    end_date = enddate,
+    sample_size = samplesize,
+    created_date = createddate
+  )
+#TODO: unsure how to deal with timestamp-- split into date and time or leave as is?
+#TODO: can I delete "president" and "modeldate" since each only has one value in this dataset (trump and Jan 5)? if so:
+trump_approval_poll <- trump_approval_poll[,-c(1,3)]
+devtools::use_data(trump_approval_poll, overwrite = TRUE)
+
+trump_approval_trend <- read_csv("data-raw/trump-approval-ratings/approval_topline.csv")
+trump_approval_trend <- trump_approval_trend %>%
+  mutate(
+    president = as.factor(president),
+    subgroup = as.factor(subgroup),
+    modeldate = as.Date(modeldate, format = "%m / %d / %Y")
+  ) %>%
+  rename(
+    approve_high = approve_hi,
+    approve_low = approve_lo,
+    disapprove_high = disapprove_hi,
+    disapprove_low = disapprove_lo
+  )
+trump_approval_poll <- trump_approval_poll[,-c(1,3)]
+devtools::use_data(trump_approval_trend, overwrite = TRUE)
+#same comments as above about timestamp, president, model date
+
+# trump-world-trust---------------------------------------------------------------------
+trumpworld_issue_1 <- read_csv("data-raw/trump-world-trust/TRUMPWORLD-issue-1.csv")
+trumpworld_issue_1 <- clean_names(trumpworld_issue_1)
+trumpworld_issue_1$country <- as.factor(trumpworld_issue_1$country)
+trumpworld_issue_1$issue <- 1
```

Comment:
Again, try to have these as a sequence of `%>%` instead of four separate options as this makes it easier to read, but also minimizes redundancy as per the [Don't Repeat Yourself](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle of coding.